releasenotes/notes/43706.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+
+issue:
+- 43705
+
+releaseNotes:
+- |
+  **Added** analyzer for telemetry resource.
```

Comment:
Include more details about what is being analyzed?

Reply:
Done